CMSgov/onemac
# OY2 20584: Remove "RAI" from "Select a Submission Type" card in Package View

Story: https://qmacbis.atlassian.net/browse/OY2-20584
Endpoint: See github-actions bot comment

### Details
Update triage choice text so that it does not suggest that you can submit RAI Responses (without parent package)

### Changes
- text changes in the triageChoices library
- updated some tests to look/match on new text

### Test Plan
1. Login as submitting user
2. Navigate to Package Dashboard
3. Click New Submission
4. Verify text of choices matches AC
5. Click Waiver Action
6. Verify text of App K matches AC
